scripts/casefold_db.py

```diff
@@ -13,27 +13,31 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+import argparse
 import json
 import sqlite3
+import sys
 from typing import Any, Dict, List, Tuple
-from tests.utils import ResolvingMemoryReactorClock
 
 import signedjson.sign
 
+from sydent.sydent import Sydent, get_config_file_path, parse_config_file
 from sydent.util import json_decoder
 from sydent.util.emailutils import sendEmail
-from sydent.sydent import Sydent, parse_config_file, get_config_file_path, parse_config_dict
 from sydent.util.hash import sha256_and_url_safe_base64
+from tests.utils import ResolvingMemoryReactorClock
+
 
 def calculate_lookup_hash(sydent, address):
-        cur = sydent.db.cursor()
-        pepper_result = cur.execute("SELECT lookup_pepper from hashing_metadata")
-        pepper = pepper_result.fetchone()[0]
-        combo = "%s %s %s" % (address, "email", pepper)
-        lookup_hash = sha256_and_url_safe_base64(combo)
-        return lookup_hash
-
-def update_local_associations(sydent, db: sqlite3.Connection):
+    cur = sydent.db.cursor()
+    pepper_result = cur.execute("SELECT lookup_pepper from hashing_metadata")
+    pepper = pepper_result.fetchone()[0]
+    combo = "%s %s %s" % (address, "email", pepper)
+    lookup_hash = sha256_and_url_safe_base64(combo)
+    return lookup_hash
+
+
+def update_local_associations(sydent, db: sqlite3.Connection, flag):
```

Comment:
I suggest we just pass the boolean flags in directly, since there are only a couple

```suggestion
def update_local_associations(sydent, db: sqlite3.Connection, send_email: bool, dry_run: bool):
```